targets/ingester/src/transform/code-du-travail.js

```diff
@@ -0,0 +1,56 @@
+import slugify from "@socialgouv/cdtn-slugify";
+import cdtnSources from "@socialgouv/cdtn-sources";
+import unistUtilSelect from "unist-util-select";
+
+import { getJson } from "../lib/getJson.js";
+
+const { SOURCES } = cdtnSources;
+const { selectAll } = unistUtilSelect;
```

Comment:
why not in the import ?

Reply:
since it was first coded as esmodule I encounter some compatibility issue (no named import, so destructuring not working)